build_config.rb

```diff
@@ -11,6 +11,7 @@ def gem_config(conf)
   conf.enable_bintest
   conf.enable_debug
   conf.enable_test
+  conf.enable_cxx_abi
```

Comment:
I just tried without it, and it failed.

http://ruby-doc.org/core-mruby/doc/guides/compile_md.html#label-C-2B-2B+ABI
> mruby can use C++ exception to raise exception internally. It is called C++ ABI mode. By using C++ exception it can release C++ stack object correctly. Whenever you mix C++ code C++ ABI mode would be enabled automatically. If you need to enable C++ ABI mode explicitly add the following: ruby conf.enable_cxx_abi 

So it should be detected automatically, moreover, I've enabled it into mruby-docopt. So it seems that it's too late when it detects and enables it.

Should `MRuby::Build` and `MRuby::CrossBuild` configuration of mrbgems infect the project depending on them?